Kentor.AuthServices/Configuration/FederationIdentityConfiguration.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.IdentityModel.Configuration;
+using System.IdentityModel.Services.Configuration;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Kentor.AuthServices.Configuration
+{
+    /// <summary>
+    /// <c>FederationIdentityConfiguration</c> is used to load a FederationConfiguration directly from Web.Config without the need to use System.Web.
+    /// </summary>
+    public class FederationIdentityConfiguration : FederationConfiguration
```

Comment:
Maybe SystemIdentityModelConfigurationsection or something that indicates that this is from System.IdentityModel and not to be confused with AuthServices' own FederationElement